boards/arm/disco_l475_iot1/disco_l475_iot1.dts

```diff
@@ -45,6 +45,76 @@
 		sw0 = &user_button;
 		eswifi0 = &wifi0;
 	};
+
+	header_pins {
```

Comment:
What about pulling all arduino definitions into a disco_l475_iot1_arduino.dtsi (at current location) than would be included in disco_l475_iot1.dts?

Reply:
I can do that.